### PR TITLE
[TASK] Make it work with TYPO3 11

### PR DESCRIPTION
**Pull request by georgringer**

### CODE REVIEW

**Comment by Stehos** [comment]
Hello, how does it look with v11 compatibility? Thank you so much.

**Comment by georgringer** [comment]
> Hello, how does it look with v11 compatibility? Thank you so much.

just use the PR or my branch, it works for us

**Review by tstahn — COMMENTED** [review]
If I'm not mistaken, removal of `ext_icon.png`requires removal of the corresponding `addPlainTextFile()` call in `ExtensionConfigurationAggregate` as well, see

https://github.com/IchHabRecht/mask_export/blob/master/Classes/Aggregate/ExtensionConfigurationAggregate.php#L127

File: Classes/Aggregate/ExtensionConfigurationAggregate.php [hunk 1]
```diff
@@ -77,7 +77,7 @@ protected function addExtEmconf()
 
         $this->addPhpFile(
             'ext_emconf.php',
-            substr($emConfUtility->constructEmConf($extensionData), 6)
+            substr($emConfUtility->constructEmConf('mask',$extensionData), 6)
```
Comment (simonschaufi) [inline Classes/Aggregate/ExtensionConfigurationAggregate.php:80]:
```suggestion
            substr($emConfUtility->constructEmConf('mask', $extensionData), 6)
```

**Comment by IchHabRecht** [comment]
Hi @georgringer,

Thank you for your pull request. After some investigation in mask 7.1 I noticed some more adjustment needed to be done. I opened a separate pull request (#189) to support mask 7.1.

**Comment by IchHabRecht** [comment]
As this pull request is incomplete, I'm closing this one.

**Comment by IchHabRecht** [comment]
As this pull request is incomplete, I'm closing this one.

**Pull request closed without merging**